python_anvil/http.py

```diff
@@ -1,20 +1,39 @@
 import requests
 from base64 import b64encode
 from logging import getLogger
-from ratelimit import limits
+from ratelimit import limits, sleep_and_retry
+from ratelimit.exception import RateLimitException
 from requests.auth import HTTPBasicAuth
 
+from python_anvil.exceptions import AnvilRequestException
+
 
 logger = getLogger(__name__)
 
-REQUESTS_LIMIT = 200
-REQUESTS_LIMIT_MS = 5000
+REQUESTS_LIMIT = {
+    "dev": {
+        "calls": 2,
+        "seconds": 1,
+    },
+    "prod": {
+        "calls": 40,
+        "seconds": 1,
+    },
+}
+
+RATELIMIT_ENV = "dev"
+
+
+def _handle_request_error(e: Exception):
+    raise e
 
 
 class HTTPClient:
-    def __init__(self, api_key=None):
+    def __init__(self, api_key=None, environment="dev"):
         self._session = requests.Session()
         self.api_key = api_key
+        global RATELIMIT_ENV  # pylint: disable=global-statement
```

Comment:
What's happening here?

Reply:
It modifies the `RATELIMIT_ENV` at the top-level of this file, otherwise it would be creating a local variable of the same name.